HangPerson/HangPerson/main.m

```diff
@@ -1,18 +1,50 @@
 //
 //  main.m
-//  HangPerson
+//  HangManGame
 //
-//  Created by Michael Kavouras on 6/15/15.
-//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//  Created by Jason Wang on 6/17/15.
+//  Copyright (c) 2015 Jason Wang. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
 
 int main(int argc, const char * argv[]) {
     @autoreleasepool {
+        // Start Code HERE
+        char hangManWord[8] = "zodiacs";
```

Comment:
you don't need to specify a size (eg. `[8]`) if you are initializing the variable with a value (eg `"zodiacs"`); You can just do: 
```c
char hangManWord[] = "zodiacs";
```